pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -43,17 +43,18 @@ import (
 )
 
 const (
-	testName              = "foo"
-	installJobName        = "foo-install"
-	uninstallJobName      = "foo-uninstall"
+	testName              = "foo-lqmsh"
+	testClusterID         = "foo"
```

Comment:
Just for clarity, I'd use here an value unrelated e.g. "bar" or "myclusterid"

Reply:
Will do.